be/src/runtime/tablets_channel.cpp

```diff
@@ -135,13 +135,12 @@ void TabletsChannel::add_chunk(brpc::Controller* cntl, const PTabletWriterAddChu
     }
 
     auto context = std::move(res).value();
-    auto channel_size = _tablet_id_to_sorted_indexes.size();
+    auto channel_size = request.has_chunk() ? _tablet_id_to_sorted_indexes.size() : 0;
     auto tablet_ids = request.tablet_ids().data();
     auto tablet_ids_size = request.tablet_ids_size();
```

Comment:
Why not return instantly after finding the PTabletWriterAddChunkRequest have no chunks?

Reply:
> Why not return instantly after finding the PTabletWriterAddChunkRequest have no chunks?

need to handle eos